fluster/decoders/gstreamer.py

```diff
@@ -900,3 +900,15 @@ def __init__(self) -> None:
         self.api = "SW"
         self.caps = self.caps + ",format=S16LE"
         super().__init__()
+        
+        
+@register_decoder
+class FluendoH264LCEVCDecoder(GStreamer10Video):
```

Comment:
Lets stick to naming similar as we have in flu-plugins -> FluendoLCEVCHWVAH264Decoder